src/hub/Hub.sol

```diff
@@ -262,6 +269,41 @@ contract Hub is Circles {
         emit InviteHuman(msg.sender, _human);
     }
 
+    /**
+     * Invite human as organization allows to register a human avatar as an organization.
+     * @param _human address of the human to invite
+     * @param _donationReceiver address of where to send the donation to with 2300 gas (using transfer)
+     */
+    function inviteHumanAsOrganization(address _human, address payable _donationReceiver) external payable {
+        require(msg.value > MINIMUM_DONATION, "Donation must be at least 0.1 xDai.");
+        require(isOrganization(msg.sender), "Only organizations can invite.");
+
```

Comment:
```suggestion
require(msg.sender != _donationReceiver, "Can't donate to yourself");
```

Reply:
Makes it at least a little bit more expensive to perpetually invite people with the same funds.

Reply:
well, I agree, but you can always send to yourself over one-hop, so I don't know if we should block it. we already know that the donation is a reputation requirement for it to have meaning

Reply:
I added the requirement with comment